scipy/stats/_stats_py.py

```diff
@@ -9858,53 +9869,106 @@ def quantile_test(x, *, q=0, p=0.5, alternative='two-sided'):
 
-        statistic1 : float
-            First test statistic: the proportion of samples in `x` that are
-            less than or equal to the hypothesized quantile `q`
-        statistic2 : float
-            Second test statistic: the proportion of samples in `x` that are
-            strictly less than the hypothesized quantile `q`
+        statistic : float
+            One of two test statistics that may be used in the quantile test.
+            The first test statistic, `T1`, is the proportion of samples in `x`
+            that are less than or equal to the hypothesized quantile `q` The
+            second test statistic, `T2`, is the proportion of samples in `x`
+            that are strictly less than the hypothesized quantile `q`.
+
+            When:
+                ``alternative = 'greater'``, `T1` is used to calculate the
+                p-value and ``statistic`` is set to `T1`.
+
+                ``alternative = 'less'``, `T2` is used to calculate the
+                p-value and ``statistic`` is set to `T2`.
+
+                ``alternative = 'two-sided'``, both `T1` and `T2` are
+                considered and the one that leads to the smallest p-value is
+                used.
+
+        statistic_type : int
+            One of `1` or `2` depending on which of `T1` or `T2` was
+            used to calculate the p-value respectively.
+
         pvalue : float
             The p-value associated with the given alternative.
 
         The object also has the following method:
 
         confidence_interval(confidence_level=0.95)
-            Computes a confidence interval around the the `p`th quantile
-            of the population. The confidence interval is returned in a
-            ``namedtuple`` with fields `low` and `high`.  Values are `nan`
-            when there are not enough samples to compute the confidence
-            interval at the desired confidence.
+            Computes a confidence interval around the the
+            :math:`p_{\mathrm{th}}` quantile of the population. The
+            confidence interval is returned in a ``namedtuple`` with
+            fields `low` and `high`.  Values are `nan` when there are
+            not enough observations to compute the confidence interval at
+            the desired confidence.
 
     Notes
     -----
     This test and its method for computing confidence intervals are
     non-parametric. They are valid if and only if the observations are i.i.d.
 
-    The implementation of the test follows Conover [1]_. The p-value of this
-    test derives from the binomial distribution but, different from the binomial
-    test, there are two different test statistics::
+    The implementation of the test follows Conover [1]_. Two test statistics
+    are considered.
 
-        k1 = (x <= q).sum()
-        k2 = (x < q).sum()
+    T1: The number of observations in `x` less than or equal to `q`.
 
-    The p-value is given by::
+        T1 = (x <= q).sum()
```

Comment:
These should render as code.

![image](https://github.com/scipy/scipy/assets/6570539/1fdba32d-197e-4476-ba03-81f43cc8867b)